conf/dev.env

```diff
@@ -0,0 +1,4 @@
+ST2_USER=sam
```

Comment:
Yes, ha! Those should have been removed, but I missed it. `$ST2_USER` and `$ST2_PASSWORD` are picked up from the host environment.